src/main.tsx

```diff
@@ -1,10 +1,13 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
-
-ReactDOM.createRoot(document.getElementById('root')!).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-)
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import { Auth0Provider } from '@auth0/auth0-react';
+ 
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <Auth0Provider
+  domain='dev-30ax000lzfhvtghw.us.auth0.com'
+  clientId='DiRAOZLjOuEFb6Q2E7OAKxTxZBCvNWJX'
+  authorizationParams={{ redirect_uri: window.location.origin }}>
```

Comment:
Zmienne środowiskowe najlepiej definiować w plikach .env ponieważ są to sekretne dane które nie mogą być dostępne dla każdego (lub i w repo). Czasami jest też tak, że na wersji developerskiej mamy inne dane do testowego autha, a na prodzie chcemy inne dane tu podpiac.

Reply:
Jest to kod wzięty z oficjalnej dokumetacji, czy w takim razie mam nauczyc sie definiować te zmienne w .env?